assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -200,6 +206,32 @@ function DashboardIdeasWidget( props ) {
 		[ setQueryParamRoute ]
 	);
 
+	// Any time the pagination value changes, scroll to the top of the container.
+	useUpdateEffect( () => {
+		const tabBarRectangle = tabBarHeaderRef?.current?.getBoundingClientRect();
+
+		if ( ! tabBarRectangle ) {
+			return;
+		}
+
+		const isOnScreen =
+			tabBarRectangle.top >= 0 &&
+			tabBarRectangle.left >= 0 &&
+			tabBarRectangle.bottom <=
+				( global.innerHeight ||
+					global?.document.documentElement.clientHeight ) &&
```

Comment:
I think we can rely on `document` always being there as we have below.
```suggestion
					global.document.documentElement.clientHeight ) &&
```